### ISSUE TITLE
Time for a new release?

### ISSUE THREAD

**Issue by jusfreeman** [veda-consulting/uk.co.vedaconsulting.mailchimp#330]
The last release was in 2017 according to the info.xml

As there's been work and bug fixes committed since 2017 it would be good to do a new release.

Agileware Ref: CIVICHIMP-5

**Comment by rollox** [comment]
Absolutely! Would be nice to have an official release so we don't need to patch the 2017 version.

**Comment by mwittmann** [comment]
There's a  thread (#305) on making this with Civi 5.x and PHP 7.x. It would be great to resolve those issues and roll them into a new release, or at least clarify the compatibility issues (and workarounds).

**Comment by twomice** [comment]
+1

**Comment by scotthue** [comment]
@deepak-srivastava +1

**Comment by veda-consulting** [comment]
@jusfreeman @rollox @twomice @scotthue 
We're picking this up - hopefully we can get a new release out soon

**Comment by mwittmann** [comment]
I'd love to see a new release. FYI, it seems that for Civi 5..x, the Mailchimp menu items are not being added to the menu structure as expected, and I've also been unable to add them manually in WordPress because the `&reset=1` query string parameter is not handled correctly. I've reported it https://civicrm.stackexchange.com/questions/34364/mailchimp-extension-menu-items-are-missing with no resolution. Can you a suggest a way I can troubleshoot & resolve this? If I can figure it out, I'd be happy to submit a pull request.

**Comment by aidenafterall** [comment]
+1
